### PR TITLE
Fixed linting issues with server code

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Mainly removing unused libraries and some style changes based on PEP8 guidelines

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
@mathcolo @idreyn Hey there's one more error in the server code regarding an unused var but wanted to check in with you guys first before removing as I don't want to break anything :)

`server/chalicelib/MbtaApi.py:12:49: F821 undefined name 'api_key'`

Any idea what's going on with this var?

**Comment by mathcolo** [comment]
@nathan-weinberg Yeaaaah that doesn't work, does it. That file is left over from a long time ago when there was something (I think) we were going to use the T's V3 API for. But in reality it can probably just be nuked from this repo for now.

File: README.md [hunk 1]
```diff
@@ -13,3 +13,16 @@ This is the repository for the TransitMatters data dashboard. Client code is wri
 2. In the root directory, run `npm install` to install all frontend and backend dependencies
 3. Run `npm start` to start both the JavaScript development server and the Python backend at the same time.
 4. Navigate to [http://localhost:3000](http://localhost:3000) (or the url provided after running `npm start`)
+
+### Linting
+To lint backend code, run the following in the root directory:
+```bash
+$ pip install flake8
```
Comment (mathcolo) [inline README.md:20]: Could this be added to the Pipfile in `server/` possibly, and then attached as a script in package.json?

File: README.md [hunk 2]
```diff
@@ -13,3 +13,16 @@ This is the repository for the TransitMatters data dashboard. Client code is wri
...
+$ flake8 --ignore=E501,E731 server
+```
+
+To lint frontend code:
+```bash
+$ npm install eslint
```
Comment (mathcolo) [inline README.md:26]: Same with eslint, maybe add as an npm dev dependency?

**Review by mathcolo — APPROVED** [review]
woo!

File: README.md (hunk 1 above)
Comment (nathan-weinberg) [reply README.md]: I think I did correctly but please double check (I am no npm expert)

**Pull request merged**